src/commands/commit/index.js

```diff
@@ -11,6 +12,7 @@ const commit = (mode: 'client' | 'hook') => {
     .then((gitmojis) => prompts(gitmojis))
     .then((questions) => {
       inquirer.prompt(questions).then((answers) => {
+        answers = escapeAnswers(answers)
```

Comment:
I'm not really convinced with this approach since it's parsing all the answers and we don't need to escape all of them. Only the ones that can receive text user input, specifically the `title`, `message` and `scope`